test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
Ah, OK, now I see. For now I'd suggest changing the error message to `type 'C1' has no member named 'subscript'` (adding the word 'named'). I guess then it's correct since `C1` doesn't have a member *named* subscript but only a subscript-member. 

In a subsequent PR we could then maybe improve the diagnostics by looking if the type would have a subscript-member (which was probably meant) and point out that selectors for subscripts are not yet supported. Once we do support them, this would turn into a fixit for the `getter` or `setter` syntax.